CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## 0.1.1 - May 4, 2020
+### Changed
```

Comment:
This also includes #4 I'm not sure if that changes the version number

Reply:
Updated. No, it doesn't change the version number.